### ISSUE TITLE
Torches disappear when lit

### ISSUE THREAD

**Issue by balkloth** [CleverRaven/Cataclysm-DDA#48729]
# Describe the bug

Torches disappear when lit.  When making a torch and lighting it by activating, the message "You light the torch." appears, but instead of wielding a lit torch, the torch disappears.

# Steps To Reproduce

1. Make a torch.
2. Light it with any fire source, including matches/lighters.
3. Get "You light the torch." message while remaining in darkness.

# Expected behavior

The torch item would be replaced with the lit torch item.

# Screenshots

![1](https://user-images.githubusercontent.com/83518210/116795717-fe230b00-aaa4-11eb-8c5d-fa9c02f3fea7.jpg)

Torch is wielded.

![2](https://user-images.githubusercontent.com/83518210/116795724-08450980-aaa5-11eb-86ee-6922afd05f4c.png)

Torch is lit.

![3](https://user-images.githubusercontent.com/83518210/116795728-109d4480-aaa5-11eb-8c85-8e009ce03ce7.png)

Torch lighting message appears.

![4](https://user-images.githubusercontent.com/83518210/116795733-1f83f700-aaa5-11eb-82ea-8e14194fdbda.png)

Torch is gone.

![5](https://user-images.githubusercontent.com/83518210/116795736-2874c880-aaa5-11eb-85bc-d33dc5b5aa6c.png)

This screen shot is of lighting the torch with matches - "You light the torch." message appears but still in darkness.

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.E-10395-g4525b4f
 - Graphics version: retrodays
 - Ingame language: English
 - Mods loaded: Default

**Comment by actual-nh** [comment]
@Salty-Panda?

**Comment by casswedson** [comment]
It is working in this build. ~5 days old.
```
- OS: Linux
    - OS Version: Distributor ID: Debian; Description: Debian GNU/Linux 10 (buster); Release: 10; Codename: buster; 
- Game Version: f17115a89e [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]
```

**Comment by actual-nh** [comment]
In other words, this can be attributed to #48642, which is being worked on (see #48705).

**Comment by casswedson** [comment]
Duplicate of #48464 fixed by #48593

**Comment by actual-nh** [comment]
Thank you.
